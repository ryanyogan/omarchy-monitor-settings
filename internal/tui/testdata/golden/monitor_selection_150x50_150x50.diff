Visual regression detected!

=== EXPECTED ===
  [90m╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m│[0m                                                               [1;34mDisplay Settings[0m                                                               [90m│[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                                                      
  [34m╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m  [1;34mMonitor Selection[0m                                                                                                                           [34m│[0m    
  [34m│[0m  [37mChoose a display to configure[0m                                                                                                               [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m  [1;32m▶ [0m[1;32mHDMI-A-1[0m [44m [0m[1;44mACTIVE[0m[44m [0m                                                                                                                         [34m│[0m    
  [34m│[0m    [37mDell U2414H[0m                                                                                                                               [34m│[0m    
  [34m│[0m    [37m1920x1080 @ 60Hz[0m                                                                                                                          [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m    [1;34mDP-1[0m [100m [0m[1;100mINACTIVE[0m[100m [0m                                                                                                                           [34m│[0m    
  [34m│[0m      [37mSamsung C27F390[0m                                                                                                                         [34m│[0m    
  [34m│[0m      [37m1920x1080 @ 75Hz[0m                                                                                                                        [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m  [33m⏎[0m[37m Select monitor and return to dashboard[0m  [35mesc[0m[37m Return to main menu[0m                                                                           [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m  [3;90m💡 Selected monitor will be marked as CURRENT on the dashboard[0m                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m│[0m                                                                                                                                              [34m│[0m    
  [34m╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                                                      
  [90m╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m│[0m                                          [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m                                        [90m│[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                                               Display Settings                                                               │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                                                      
  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │  Monitor Selection                                                                                                                           │    
  │  Choose a display to configure                                                                                                               │    
  │                                                                                                                                              │    
  │  ▶ HDMI-A-1  ACTIVE                                                                                                                          │    
  │    Dell U2414H                                                                                                                               │    
  │    1920x1080 @ 60Hz                                                                                                                          │    
  │                                                                                                                                              │    
  │    DP-1  INACTIVE                                                                                                                            │    
  │      Samsung C27F390                                                                                                                         │    
  │      1920x1080 @ 75Hz                                                                                                                        │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │  ⏎ Select monitor and return to dashboard  esc Return to main menu                                                                           │    
  │                                                                                                                                              │    
  │  💡 Selected monitor will be marked as CURRENT on the dashboard                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                                                      
  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                          ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                                        │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 10748
Actual length: 9612
Expected hash: 49325dfaed031da2d59849357c45c8bb449a159d24513f8afc3e0a14900f48f8
Actual hash: 926c8cfa5487c7cd4b135224641f7e52e5aa155b5ef86fe337359020603d1e35
