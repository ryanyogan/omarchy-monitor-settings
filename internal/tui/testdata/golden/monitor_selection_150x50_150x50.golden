# Visual Golden File
# Name: monitor_selection_150x50
# Dimensions: 150x50
# Hash: 926c8cfa5487c7cd4b135224641f7e52e5aa155b5ef86fe337359020603d1e35

  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                                               Display Settings                                                               │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                                                      
  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │  Monitor Selection                                                                                                                           │    
  │  Choose a display to configure                                                                                                               │    
  │                                                                                                                                              │    
  │  ▶ HDMI-A-1  ACTIVE                                                                                                                          │    
  │    Dell U2414H                                                                                                                               │    
  │    1920x1080 @ 60Hz                                                                                                                          │    
  │                                                                                                                                              │    
  │    DP-1  INACTIVE                                                                                                                            │    
  │      Samsung C27F390                                                                                                                         │    
  │      1920x1080 @ 75Hz                                                                                                                        │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │  ⏎ Select monitor and return to dashboard  esc Return to main menu                                                                           │    
  │                                                                                                                                              │    
  │  💡 Selected monitor will be marked as CURRENT on the dashboard                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                                                      
  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                          ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                                        │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    